socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -165,6 +167,29 @@ def test_redact_raw_crash(self):
         eq_(crash, expected_crash)
 
 
+class TestIsValidKey(object):
+    @pytest.mark.parametrize('key', [
+        'a',
+        'abc',
+        'ABC',
+        'AbcDef',
+        'Abc_Def',
+        'Abc-def'
+        'Abc-123-def'
+    ])
+    def test_valid_key(self, key):
+        assert is_valid_key(key) is True
+
+    @pytest.mark.parametrize('key', [
+        '',
+        '.',
+        'abc def',
+        u'na\xefve',
```

Comment:
Three of these are cases @milescrabill's script deals with. The one with a space in it I added because it seemed like an interesting case.

@peterbe, @milescrabill: Are there other known-bad cases this is missing that we should add?